Source/OxyPlot/Axes/Rendering/AngleAxisRenderer.cs

```diff
@@ -51,13 +51,18 @@ public override void Render(Axis axis, int pass)
 
             var axisLength = Math.Abs(scaledEndAngle - scaledStartAngle);
             var eps = axis.MinorStep * 1e-3;
+
+            ScreenPoint someScreenPoint = ScreenPoint.Undefined;
```

Comment:
This could do with a better name

Reply:
maybe

`ScreenPoint innerRadiusTangentScreenPoint = ScreenPoint.Undefined;`

Reply:
`innermostPoint` would make sense; may need to check if reversing the axis messes with this (e.g. setting `StartPosition=1` and `EndPosition=0`)

Reply:
Is it not the case that all points  of the angle axis have the same distance to the center? Am I getting it wrong right now?

Reply:
Ah, yes of course: sorry, my mistake.

Reply:
Given the points are all on the circumference, can we just derive one using `magnitudeAxis.Transform(magnitudeAxis.ClipMaximum, <some sensible value>, axis))`?

Actually, that inspires a name: `pointOnInnerCircumference` (long, but explicit, and hopefully should be clear if I've missed the point)